app/kitchensink/kitchensink.js

```diff
@@ -31,6 +31,66 @@ $(document).ready(function () {
         }, 1000);
       });
 
+      //adding ellipsis to the large text
+      var ellipsisText = function (e) {
+        //complete text stored in a variable
+        var completeText = e.innerHTML.split(" ");
+        //wordArray will be the text displayed
+        var wordArray = e.innerHTML.split(" ");
+        while (e.scrollHeight > e.offsetHeight) {
```

Comment:
Would clientHeight work better?

Reply:
I can try, but doesn't seem like it, as the height is always changing based on the class used in the card.